lib/glossarist.rb

```diff
@@ -30,6 +31,7 @@
 
 require_relative "glossarist/collections"
 
+require_relative "glossarist/lutaml_models"
```

Comment:
@HassanAkbar why do we have to use a special namespace for these? Why not just replace the classes in the original namespace?